classes/class-wc-connect-shipping-label.php

```diff
@@ -283,14 +283,23 @@ protected function get_form_data( WC_Order $order ) {
 			$origin                 = $this->get_origin_address();
 			$selected_rates         = $this->get_selected_rates( $order );
 			$destination            = $this->get_destination_address( $order );
+			$unpacked = array();
+
+			// if there's no packaging information, add everything to the unpacked list
+			if ( ! $this->get_packaging_metadata( $order ) ) {
+				foreach( $selected_packages as $package ) {
+					$unpacked[] = $package[ 'items' ][ 0 ];
+				}
+				$selected_packages = new stdClass();
+			}
```

Comment:
That's definitely the right way to do it. I cut that corner to save some time but if you think it's worth the refactor I can do that.